recipe/meta.yaml

```diff
@@ -23,14 +24,18 @@ requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
 
 test:
   requires:
-    - ipaddress    # [py<33]
-    - mock         # [py<33]
-    - unittest2    # [py<27]
+    - ipaddress    # [py<30]
+    - mock         # [py<30]
+    - unittest2    # [py<30]
+    - enum34       # [py<35]
+    - pip
 
   imports:
     - psutil
```

Comment:
Private modules with `_modulename` are better to remove:
```
    - psutil
    - psutil.tests
```

Reply:
The the private modules were removed:

https://github.com/AnacondaRecipes/psutil-feedstock/blob/pbs_5.9.0_20211230T170103/recipe/meta.yaml#L36&&L37